Sources/Extensions/UIKit/UILayoutPriorityExtensions.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UILayoutPriorityExtensions.swift
+//  SwifterSwift
+//
+//  Created by diamantidis on 8/19/18.
+//  Copyright © 2018 SwifterSwift. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+import UIKit
+
+// MARK: - Initializers
```

Comment:
The MARK should be inside the extension, before the functions.

Reply:
Hi @guykogus, it is removed.
Initially I added the MARK comment there in order to keep it consistent with the other extensions. 
Most of them, if not all, follow this pattern. And also there is a section about the MARK comments [here](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md#adding-new-extensions)